code/datums/computerfiles.dm

```diff
@@ -107,6 +107,8 @@
 
 
 	file
+		/// Prevent it from piracy via PDA filehosting?
+		var/dont_copy = 0
```

Comment:
Imo trying to copy a program with `dont_copy == 1` should again produce an error message. Or the `Copy` button could just be grayed out for these.

Reply:
Good idea! If a file can't be copied, the link is replaced with a strikethrough, now.